assets/src/dashboard/components/card-grid-item/card-preview.js

```diff
@@ -0,0 +1,131 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { BUTTON_TYPES } from '../../constants';
+import { Button } from '../';
+
+const PreviewImage = styled.img`
+  object-fit: cover;
+  border-radius: 8px;
+  width: 100%;
+  height: ${({ theme }) => theme.grid.desktop.imageHeight};
+
+  @media ${({ theme }) => theme.breakpoint.tablet} {
+    height: ${({ theme }) => theme.grid.tablet.imageHeight};
+  }
+
+  @media ${({ theme }) => theme.breakpoint.mobile} {
+    height: ${({ theme }) => theme.grid.mobile.imageHeight};
+  }
+
+  @media ${({ theme }) => theme.breakpoint.min} {
+    height: ${({ theme }) => theme.grid.min.imageHeight};
+  }
+`;
+
+const EditControls = styled.div`
+  width: ${({ theme }) => theme.grid.desktop.itemWidth};
+  height: ${({ theme }) => theme.grid.desktop.imageHeight};
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @media ${({ theme }) => theme.breakpoint.tablet} {
+    height: ${({ theme }) => theme.grid.tablet.imageHeight};
+    width: ${({ theme }) => theme.grid.tablet.itemWidth};
+  }
+
+  @media ${({ theme }) => theme.breakpoint.mobile} {
+    height: ${({ theme }) => theme.grid.mobile.imageHeight};
+    width: ${({ theme }) => theme.grid.mobile.itemWidth};
+  }
+
+  @media ${({ theme }) => theme.breakpoint.min} {
+    height: ${({ theme }) => theme.grid.min.imageHeight};
+    width: ${({ theme }) => theme.grid.min.itemWidth};
+  }
+`;
+
+const PreviewButton = styled(Button)`
+  font-size: 22px;
+  align-self: center;
+  flex-grow: 1;
+  line-height: 31px;
+`;
+
+const CtaButton = styled(Button)`
+  width: calc(100% - 32px);
+  height: 40px;
+  margin-bottom: 25px;
+`;
+
+// TODO modify to handle other types of grid items, not just own stories
+const CardPreviewContainer = ({
+  onOpenInEditorClick,
+  onPreviewClick,
+  previewSource,
+}) => {
+  const displayEditControls = onPreviewClick || onOpenInEditorClick;
+
+  return (
+    <>
+      <PreviewImage src={previewSource} alt="preview" />
+      {displayEditControls && (
+        <EditControls>
+          {onPreviewClick && (
+            <PreviewButton
+              type={BUTTON_TYPES.SECONDARY}
+              onClick={onPreviewClick}
+            >
+              {__('Preview', 'web-stories')}
+            </PreviewButton>
```

Comment:
It looks like the preview's button clickable area is a big pill (I applied a red border to it in the screenshot below).  I feel it should either be a rectangle that encompasses the entire above "Open in editor" area, or maybe the entire story card itself should be clickable to open up the preview 🤔 What do you think?

![image](https://user-images.githubusercontent.com/40646372/77667444-71238f00-6f3f-11ea-910e-98c8ac831264.png)

Reply:
oh good call out! honestly, I spent very little time or attention on the actual functionality of the eventual "hover". I think it should just be the space of the normal button for now and then we should get @samitron7 's take on it - since the open in editor button would overlap with the preview if it's the entire space of the rectangle. 

regardless, that weird oval has got to go.

Reply:
<img width="210" alt="Screen Shot 2020-03-26 at 10 58 11 AM" src="https://user-images.githubusercontent.com/10720454/77680963-eac47880-6f51-11ea-9f51-9a100acad186.png">

Reply:
Coolio, yeah I'd be curious of Sam's opinion on this.  I was under the impression that the base card would be clickable to trigger opening up the preview window, and then `Open in editor` and the bookmark button would sit on top of the card z-index wise, so clicking on them wouldn't trigger a preview, but clicking anywhere else would.  Buuuuut, that's easily refactorable later if need be, so I'm gonna approve!

Reply:
yeah i think a convo about this interaction will need to take place - there's variations on what this looks like by view too. Easier when we're building out the actual view probs.